src/sentry/static/sentry/app/views/issueList/savedSearchMenu.tsx

```diff
@@ -52,6 +57,10 @@ function SavedSearchMenu({
             <MenuItemLink tabIndex={-1} onClick={() => onSavedSearchSelect(search)}>
               <SearchTitle>{search.name}</SearchTitle>
               <SearchQuery>{search.query}</SearchQuery>
+              <SearchSort>
+                {t('Sort By: ')}
+                {getSortLabel(search.sort)}
```

Comment:
may need to change the text color if the menu item is selected as the contrast is really low on the sort by text at the end here:

![image](https://user-images.githubusercontent.com/9372512/108142931-94858a00-707b-11eb-88e2-8fdbe5922e3b.png)